scout/server/templates/report_base.html

```diff
@@ -6,6 +6,12 @@
 	{% block css %}
 	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
 	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+
+	<style>
```

Comment:
Looks like this:

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/28093618/211315154-97367abb-74ec-4f06-8dd4-2b5c7cff6749.png">